code/modules/admin/verbs/debug.dm

```diff
@@ -743,6 +743,14 @@ ADMIN_VERB(reestablish_tts_connection, R_DEBUG, "Re-establish Connection To TTS"
 	message_admins("[key_name_admin(user)] successfully re-established the connection to the TTS HTTP server.")
 	log_admin("[key_name(user)] successfully re-established the connection to the TTS HTTP server.")
 
+ADMIN_VERB(allow_browser_inspect, R_DEBUG, "Allow Browser Inspect", "Allow browser debugging via inspect", ADMIN_CATEGORY_DEBUG)
+	if(user.byond_version < 516)
+		to_chat(user, span_warning("You can only use this on 516!"))
+		return
+
+	to_chat(user, span_notice("You can now right click to use inspect on browsers."))
+	winset(user, null, list("browser-options" = "devtools,find,refresh,byondstorage"))
```

Comment:
Probably better to use the +devtools format so the rest is kept as before

Reply:
I'm not sure I doubt it will work reliably, if at all. Especially considering that there are no examples of such use anywhere
Also, there is no mention of this use in DMReference